wdio/test-runner/typescript/package.json

```diff
@@ -15,7 +15,7 @@
     "@wdio/mocha-framework": "^8.11.0",
     "@wdio/spec-reporter": "^8.11.2",
     "chai": "^4.3.7",
-    "chromedriver": "^122.0.4",
+    "chromedriver": "^126.0.4",
```

Comment:
we should make a tracking ticket to use browser-driver-manager for this, as-is it'll break in a few months

Reply:
Ref: https://github.com/dequelabs/watcher-examples/issues/197